packages/@uppy/dashboard/src/Dashboard.tsx

```diff
@@ -250,12 +252,18 @@ export default class Dashboard<M extends Meta, B extends Body> extends UIPlugin<
 
     this.defaultLocale = locale
 
-    // Dynamic default options:
-    this.opts.doneButtonHandler ??= () => {
```

Comment:
Was this supposed to be deleted?

Reply:
wtf no, thanks for noticing, will remerge